/**/thrust_cmd_relay:
  ros__parameters:
    input_topic: 'core/thruster_cmd_float'
    output_topic1: 'core/thruster1_cmd'
    output_topic2: 'core/thruster2_cmd'

/**/thrust_vector_relay:
  ros__parameters:
    vertical_angle_topic: 'core/thrust_vector_cmd_vertical'
    horizontal_angle_topic: 'core/thrust_vector_cmd_horizontal'
    thruster_angles_topic: 'core/thrust_vector_cmd'